src/rdkafka_queue.c

```diff
@@ -587,6 +616,10 @@ int rd_kafka_q_serve_rkmessages (rd_kafka_q_t *rkq, int timeout_ms,
                                        RD_KAFKA_Q_CB_RETURN, NULL);
                 if (res == RD_KAFKA_OP_RES_KEEP ||
                     res == RD_KAFKA_OP_RES_HANDLED) {
+                    if(rko->rko_type == RD_KAFKA_OP_BARRIER) {
+                        // For EAGER rebalancing, clean the whole buffer
```

Comment:
Use /**/ comments.
The comment is not strictly related to EAGER rebalancing, but rather that the partition version changed.